gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -378,9 +387,15 @@ private Composite createStage(Composite parent, API.Stage currentStage) {
               withLayoutData( createLink(valueComposite,"<a>" + dv.displayValue + "</a>", e -> models.follower.onFollow(dv.link)),
                   new GridData(SWT.LEFT, SWT.CENTER, true, true));
             } else {
-              Label valueLabel  = withLayoutData( createLabel(valueComposite, dv.displayValue),
+              Label valueLabel = withLayoutData( createLabel(valueComposite, dv.displayValue),
                   new GridData(SWT.LEFT, SWT.CENTER, true, true));
-              valueLabel.setToolTipText(dv.tooltipValue);
+
+              if (kvp.getActive()) {
+                valueLabel.setToolTipText(kvp.getDependee().equals("") ? "Activated by " + kvp.getDependee() : dv.tooltipValue);
```

Comment:
Something is broken here somewhere. Looks like all active KVP values now have a "Activated by" tooltip.

Reply:
The ternary was flipped. Should be fixed now.

Reply:
Seems to be fixed